templates/monitoring/structured_detailed_error_counts_v1/view.sql

```diff
@@ -0,0 +1,41 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.pipeline.structured_detailed_error_counts_v1`
+AS
+WITH error_examples AS (
+  SELECT
+    TIMESTAMP_TRUNC(submission_timestamp, HOUR) AS hour,
+    document_namespace,
+    document_type,
+    document_version,
+    error_type,
+    error_message,
+    udf_js_gunzip(ANY_VALUE(payload)) AS sample_payload,
```

Comment:
```suggestion
    udf_js.gunzip(ANY_VALUE(payload)) AS sample_payload,
```

Cannot use a temporary UDF in a view.